### PR TITLE
Feature clean up shooter code

### PR DESCRIPTION
**Pull request by Tr01ler**
):

### CODE REVIEW

**Comment by Tr01ler** [comment]
I don't know what's happening, but I haven't even made any changes yet.

**Comment by Beckoe** [comment]
Everything looks good, but we need a change now. Please run the shooter motors separately (so one doesn't follow the other) and make one side run at a slightly lower speed. Use the faster side in getShooterMotorPower(). Also, move all values to a constants file (voltage, motor speeds, etc.).

File: RobotCode/src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 1]
```diff
@@ -26,34 +27,33 @@ public ShooterSubsystem()
...
 
 		// Initializes motor object
 		ShooterMotorMain = new TeamSparkMAX("Left Shooter Motor Main", Ports.SHOOTER_MOTOR_MAIN);
 		ShooterMotorSub = new TeamSparkMAX("Right Shooter Motor Main", Ports.SHOOTER_MOTOR_SUB);
+
 		FeederMotorMain = new TeamSparkMAX("Left Shooter Motor Sub", Ports.FEEDER_MOTOR_MAIN);
 		FeederMotorSub = new TeamSparkMAX("Right Shooter Motor Sub", Ports.FEEDER_MOTOR_SUB);
 
 		// Sets the motors voltage compensation
 		ShooterMotorMain.enableVoltageCompensation(VOLTAGE);
```
Comment (renatodellosso) [inline RobotCode/src/main/java/frc/robot/subsystems/ShooterSubsystem.java:43]: Voltage comp is done by default now, you don't need to do it here

File: RobotCode/src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 2]
```diff
@@ -63,27 +63,38 @@ public double getShooterMotorPower()
 		return ShooterMotorMain.get();
 	}
 
-	/* This is clamping the shooter motor power to be within the range of -1 to 1 */
+	/**
+	 * This is clamping the shooter motor power to be within the range of -1 to 1
+	 * 
+	 * @param power
```
Comment (renatodellosso) [inline RobotCode/src/main/java/frc/robot/subsystems/ShooterSubsystem.java:69]: Remove the @param parameter name if you haven't added any detail to it

**Review by renatodellosso — CHANGES_REQUESTED** [review]

File: RobotCode/src/main/java/frc/robot/subsystems/ShooterSubsystem.java [hunk 3]
```diff
@@ -63,27 +56,35 @@ public double getShooterMotorPower()
...
-	/* Continuously updates shooter speed based on the commands above. */
+	/**
+	 * Continuously updates shooter speed based on the commands above.
+	 */
 	@Override
 	public void periodic()
 	{
 		double newShooterPower = shooterPid.calculate(ShooterMotorSub.get(), shooterMotorPower);
 		ShooterMotorMain.set(newShooterPower);
+		ShooterMotorSub.set(newShooterPower * 0.8);
```
Comment (renatodellosso) [inline RobotCode/src/main/java/frc/robot/subsystems/ShooterSubsystem.java:84]: Make the 0.8 a constant

**Review by renatodellosso — CHANGES_REQUESTED** [review]

**Review by renatodellosso — APPROVED** [review]
Great work!

**Pull request merged**